Rakefile

```diff
@@ -1,14 +1,16 @@
+require "json"
+
 def e(cmd)
   puts cmd
-  system(cmd) || abort("Error running `#{cmd}`")
+  system(cmd) || raise("Error running `#{cmd}`")
```

Comment:
so it can be caught instead of exiting the program. I also think if any of the other commands were to encounter an error here the stack trace information would be useful